django/core/checks/database.py

```diff
@@ -9,3 +11,23 @@ def check_database_backends(*args, **kwargs):
     for conn in connections.all():
         issues.extend(conn.validation.check(**kwargs))
     return issues
+
+
+@register(Tags.database)
+def check_migration_modules(*args, **kwargs):
+    from django.conf import settings
+    migration_modules = getattr(settings, 'MIGRATION_MODULES', {})
+    warnings = []
+    for app_label, module in migration_modules.items():
```

Comment:
the keys are not necessarily app labels but might as well be paths to app configs. Maybe replace `app_label` with `app`?